simapp/simd/cmd/root.go

```diff
@@ -306,6 +307,7 @@ func newApp(
 		baseapp.SetSnapshot(snapshotStore, snapshotOptions),
 		baseapp.SetIAVLCacheSize(cast.ToInt(appOpts.Get(server.FlagIAVLCacheSize))),
 		baseapp.SetIAVLDisableFastNode(cast.ToBool(appOpts.Get(server.FlagDisableIAVLFastNode))),
+		baseapp.SetMempool(mempool.NewSenderNonceMempool(mempool.SenderNonceMaxTxOpt(cast.ToInt(appOpts.Get(server.FlagMempoolMaxTx))))),
```

Comment:
If this is unset does simapp start up with max-tx = 0?

Reply:
It should start with 100000 I think

Reply:
I don't think so, looks likes it being set from the value of `(appOpts.Get(server.FlagMempoolMaxTx))` which is what by default?